client/components/domains/use-my-domain/transfer-or-connect/style.scss

```diff
@@ -166,6 +171,7 @@
 				.button {
 					padding: 8px 39px;
 					flex-grow: 1;
+					white-space: nowrap;
```

Comment:
> Prevents "Purchase a plan" button text from wrapping (this should work with whatever length because the text before it is adaptable)

Also affects the blue Select button. 

<img width="292" alt="Screenshot 2025-03-12 at 11 43 21 am" src="https://github.com/user-attachments/assets/3797acd3-3312-42b2-a147-2413018af106" />

Not an issue, and good for consistency, just saying!